.gitignore

```diff
@@ -40,3 +40,4 @@ base_layer/wallet_ffi/.cargo/config
 keys.json
 node_modules
 /integration_tests/temp
+comms/dht/file
```

Comment:
Is this needed?

Reply:
It looks like it was created after running tests and not removed the end.

Reply:
Yeah this is only an issue on windows with in memory databases on sqlite - on *nix no files are created. So makes sense to ignore it